src/test/java/dev/openfeature/javasdk/ProviderSpecTest.java

```diff
@@ -54,16 +55,6 @@ public class ProviderSpecTest {
         assertNull(result.getErrorCode());
     }
 
-    @Specification(number="2.8", text="In cases of abnormal execution, the provider MUST indicate an " +
-            "error using the idioms of the implementation language, with an associated error code having possible " +
-            "values PROVIDER_NOT_READY, FLAG_NOT_FOUND, PARSE_ERROR, TYPE_MISMATCH, or GENERAL.")
-    @Disabled("I don't think we expect the provider to do all the exception catching.. right?")
-    @Test void error_populates_error_code() {
-        AlwaysBrokenProvider broken = new AlwaysBrokenProvider();
-        ProviderEvaluation<Boolean> result = broken.getBooleanEvaluation("key", false, new EvaluationContext(), FlagEvaluationOptions.builder().build());
-        assertEquals(ErrorCode.GENERAL, result.getErrorCode());
-    }
-
```

Comment:
This was basically testing a provider impl, which we don't need to do. I've deleted this test.